main.js

```diff
@@ -1,10 +1,10 @@
-var header = document.getElementsByTagName('header')[0];
-var form = document.getElementsByTagName('form')[0];
-var p = document.getElementsByClassName('d-none')[0];
-var currentID;
-var currentData;
-var gradeform = new GradeForm(form);
-var pageheader = new PageHeader(header);
-var gradetable = new GradeTable(table, p);
-var app = new App(gradetable, pageheader, gradeform);
+const header = document.getElementsByTagName('header')[0];
+const form = document.getElementsByTagName('form')[0];
+const p = document.getElementsByClassName('d-none')[0];
```

Comment:
Use query selector instead. It is much more efficient since you dont have to query the whole dom once you have found your element.